test/lackd/t_entry.clj

```diff
@@ -0,0 +1,46 @@
+(ns lackd.t-entry
+  (:require [midje.sweet :refer :all]
+            [lackd.entry :as entry]
+            [taoensso.nippy :as nippy])
+  (:import [com.sleepycat.je DatabaseEntry]))
+
+;;; :stress-record :date :lazy-seq-empty :true :long :double :lazy-seq :short
+;;; :meta :str-long :bigint :sym-ns :queue :float :sorted-set :nested
+;;; :queue-empty :lotsa-small-keywords :map-empty :set-empty :false :list-empty
+;;; :vector :kw :sym :str-short :ex-info :integer :list :ratio :byte :bigdec
+;;; :nil :sorted-map :bytes :regex :exception :uuid :set :list-quoted
+;;; :throwable :vector-empty :lotsa-small-strings :kw-ns :map
+;;; :lotsa-small-numbers :char
+(def stress-data
+  (dissoc nippy/stress-data
+          ;; Unsupported by data.fressian
+          :queue :queue-empty :byte
+          :ex-info :exception :throwable
+          ;; Supported by data.fressian, but could not to compare directly
+          :bytes :regex))
```

Comment:
I fixed it by https://github.com/ayamada/lackd/commit/1c646edc6190a728dc5c8731d7d3e2bc7b4028a5 .